crates/ibc/src/clients/ics07_tendermint/consensus_state.rs

```diff
@@ -172,14 +171,15 @@ mod tests {
     #[test]
     fn serialization_roundtrip_no_proof() {
         let json_data =
-            include_str!("../../../tests/support/query/serialization/consensus_state.json");
+            include_str!("../../../../ibc-testkit/src/utils/dummies/json/consensus_state.json");
```

Comment:
is it okay to move the `json` files as following
```
ibc-testkit/src/data/json/dummy_consensus_state.json
```

also, I don't like relative path as this is relative to the current file - which makes the refactoring messy. can we use `concat!(env!("CARGO_MANIFEST_DIR"), "/../ibc-testkit/.....")`?

Reply:
Good point: [15a01d1](https://github.com/cosmos/ibc-rs/pull/954/commits/15a01d17c219c68ecd4388577c5ae14e3ed78e1b)

Reply:
On a second thought, it is better to keep the dummy JSON files under `ibc` project directly, as it is the only place it is being used. Anyway, we aren't exporting the dummy JSON files in `ibc-testkit`.

But keep them under `/ibc/tests/dummies/*.json`.

Reply:
Also, we should use the term `fixture` instead of `dummy`. Maybe in future PR. :wink:

Reply:
> Also, we should use the term fixture instead of dummy. Maybe in future PR. 😉

Those are the inherited naming conventions, yes. better to address in a separate PR.